relay-server/src/services/store.rs

```diff
@@ -1385,8 +1388,8 @@ struct SpanKafkaMessage<'a> {
     span_id: &'a str,
     #[serde(default)]
     start_timestamp_ms: u64,
-    #[serde(default, skip_serializing_if = "Option::is_none")]
-    tags: Option<&'a RawValue>,
+    #[serde(skip_deserializing, skip_serializing_if = "BTreeMap::is_empty")]
+    tags: BTreeMap<String, String>,
```

Comment:
According to [docs](https://serde.rs/field-attrs.html#skip_serializing):

> Skip this field when serializing, but not when deserializing.

Is this intentional?

Reply:
You linked the documentation to `skip_serializing`, which I'm not using.

My intention is to not deserialize this field at all and only serialize it if it's not empty, and that's what this annotation will do I believe.

Reply:
Oh right, misread :D